contracts/secret-storage/TeeManager.sol

```diff
@@ -554,7 +558,19 @@ contract TeeManager is
     ) internal {
         uint256 stakeAmount = teeNodes[_enclaveAddress].stakeAmount;
         // compounding slashing formula: remainingStakeAmount = stakeAmount * (1 - (r/100)) ^ n
-        uint256 remainingStakeAmount = stakeAmount * ((SLASH_MAX_BIPS - SLASH_PERCENT_IN_BIPS) ** _missedEpochsCount) / (SLASH_MAX_BIPS ** _missedEpochsCount);
+        // uint256 remainingStakeAmount = stakeAmount * ((SLASH_MAX_BIPS - SLASH_PERCENT_IN_BIPS) ** _missedEpochsCount) / (SLASH_MAX_BIPS ** _missedEpochsCount);
+        
+        uint256 remainingStakeAmount = stakeAmount;
+        uint256 iterations = _missedEpochsCount / 12;
```

Comment:
value 12 can be stored in a constant. And add comment to that constant definition upon how the 12 is calculated.